CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+* @CoveMB @emnul @MCarlomagno @shahnami @tirumerla @zeljkoX
+SECURITY.md @emnul @MCarlomagno @tirumerla @zeljkoX
```

Comment:
This is the list based on contributors, we can add others going forward. To begin with i added this.